vumi/scripts/vumi_model_migrator.py

```diff
@@ -97,7 +108,8 @@ def migrate_key(self, key, dry_run):
         try:
             obj = yield self.model.load(key)
             if obj is not None:
-                if obj.was_migrated and not dry_run:
+                should_save = yield self.post_migrate_function(obj)
+                if should_save and not dry_run:
```

Comment:
I noticed, but I would've thought that we only want to run the post-migrate function on objects that have been migrated. Seems a bit surprising that the post-migrate function would run on every object, regardless of whether or not it has been migrated.